File: java/Gate One/taskTen.java
import java.util.Scanner;

public class taskTen{
    public static void main(String[] args){
        Scanner userInput = new Scanner(System.in);
	System.out.print("Enter a number: ");
	int firstNumber = userInput.nextInt();
	System.out.print("Enter a number: ");
	int secondNumber = userInput.nextInt();
	System.out.print("Enter a number: ");
	int thirdNumber = userInput.nextInt();
	
	float average = (firstNumber + secondNumber + thirdNumber) / 3;
	System.out.printf("The average of your three numbers are %.2f \n", average);
    }
}

File: javaScript/Gate One/TaskTen.js
const input = require("prompt-sync")(); 

let firstNumber = input("Enter a number ");
let secondNumber = input("Enter a number ");
let thirdNumber = input("Enter a number ");
let average = (+firstNumber + +secondNumber + +thirdNumber) / 3

console.log(`The average of your three numbers are ${average}`);
